hack/dev-env/deploy.sh

```
@@ -51,6 +51,9 @@ cleanup() {
 trap cleanup EXIT
 
 cp -a ${BASEPATH}/install/kubernetes/* ${TMPDIR}
+cp -a ${BASEPATH}/install/kubernetes/agent ${TMPDIR}/agent-managed
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)